kafka-streams-framework/src/test/java/org/hypertrace/core/kafkastreams/framework/rocksdb/BoundedMemoryConfigSetterTest.java

```diff
@@ -41,6 +41,7 @@ public void setUp() {
     RocksDBCacheProvider.get().testDestroy();
     options = new Options();
     configs = new HashMap<>();
+    configs.put("application.id", "test-app");
```

Comment:
if possible, please add a separate test for multiple appliactaions

Reply:
parameterized the tests and tested for multiple application ids